dist/gaia-header.js

```diff
@@ -913,11 +918,40 @@ module.exports = component.register('gaia-header', {
   observerStop: function() {
     if (!this.observing) { return; }
     this.observer.disconnect();
+    window.removeEventListener('resize', this);
+    this._resizeHandled = false;
+
     this.observing = false;
     debug('observer stopped');
   },
 
   /**
+   * Handle DOM events
+   */
+  handleEvent: function(e) {
+    switch(e.type) {
+      case 'resize':
+        this.onResize();
+        break;
+    }
+  },
+
+  /**
+   * handle 'resize' events
+   */
+  onResize: function() {
+    if (this._resizeHandled) {
+      return;
+    }
+
+    this._resizeHandled = true;
```

Comment:
a change I'll do: I'll keep the result of requestAnimationFrame (instead of a boolean) so that I can clear it in clearPending. Is it ok for you ?